modules/users/manifests/conorglynn.pp

```diff
@@ -1,8 +0,0 @@
-# Creates conorglynn user
-class users::conorglynn {
-  govuk_user { 'conorglynn':
-    fullname => 'Conor Glynn',
```

Comment:
Can we `ensure => absent` prior to removing the manifest file, as per [the docs on removing users](https://docs.publishing.service.gov.uk/manual/removing-a-user-from-puppet.html)?

Reply:
cool thanks, did know above this docs, was doing freestyle